docs/content/guides/cell-types/numeric-cell-type.md

```diff
@@ -11,10 +11,14 @@ canonicalUrl: /numeric-cell-type
 
 ## Overview
 
-By default, Handsontable treats all cell values as `string` type. This is because the `<textarea>` returns a string as its value. There are many cases where you need cell values to be treated as a `number` type. The numeric cell type allows you to format displayed numbers nicely and sort them correctly.
+By default, Handsontable treats all cell values as `string` type. This is because the text editor returns a string as its value. There are many cases where you need cell values to be treated as a `number` type. The numeric cell type allows you to format displayed numbers nicely and sort them correctly.
 
 ## Usage
 
+::: tip
+Ensure your numeric cell values are stored numbers and not strings in the data source, as Handsontable doesn't parse strings to numbers.
```

Comment:
♥

Reply:
Frankly, this text isn't new. I just reverted the text from before https://github.com/handsontable/handsontable/pull/9701, because I couldn't not agree with it.